modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +63,16 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const transferAppropriateAttributes = <T extends HTMLElement> (original: T, clone: T): void => {
```

Comment:
`SugarElement` here instead of raw dom elements please, as you're using sugar elements previously.

Reply:
Also for consistency, this should probably be called `cloneAppropriateAttributes` (to match `cloneCss`).

Reply:
Why did you got from `transfer` -> `copy`? That's still not consistent.